doc/src/tutorial/matrices.rst

```diff
@@ -470,10 +470,12 @@ by injecting a custom zero test with warnings enabled.
     __main__:9: UserWarning: Zero testing of -(4*cosh(q/3)**2 - 1)*exp(-q) - 2*cosh(q/3) - exp(-q) evaluated into None
     []
 
-In this case, ``(-exp(q) - 2*cosh(q/3))*(-2*cosh(q/3) - exp(-q)) - (4*cosh(q/3)**2 - 1)**2`` should yield zero,
-but the zero testing had failed to catch.
+In this case,
+ ``(-exp(q) - 2*cosh(q/3))*(-2*cosh(q/3) - exp(-q)) - (4*cosh(q/3)**2 - 1)**2``
```

Comment:
Should a space be removed from the beginning?

Reply:
@jksuom 
I have fixed